src/web/core/src/main/java/org/geoserver/web/util/MetadataMapModel.java

```diff
@@ -29,11 +29,7 @@
 
     protected Class<?> target;
 
-    protected Serializable value;
-
-    public MetadataMapModel(MetadataMap map, String expression, Class<?> target) {
-        this(new MetadataMapWrappingModel(map), expression, target);
-    }
+    protected transient Serializable value;
```

Comment:
Good point. 
The reason I removed it is because this design does not appear to be compatible with hazelcast. After serialization and recreation of the model, it will contain an object that is no longer connected to its original owner. Nothing will be saved. This is not the case if you create it from an existing model, like below.

This page appeared to be the only place in the geoserver code base that is using it. That is also why this is the only page in the UI that does not work with hazelcast.

However, I didn't consider that third parties may be using it. Should I perhaps instead deprecate it with a warning?

Reply:
Yes, best to deprecate, make sure nothing calls it, but leave it for extensions outside GeoServer.